src/styles/sass/new-design.scss

```diff
@@ -14,10 +13,10 @@
   border-top: 1px solid $white;
 }
 .sovryn-border-bottom {
-  border-bottom: 5px solid $white;
+  border-bottom: 1px solid $white;
```

Comment:
We are removing big borders from design?

Reply:
I'll check with Tzu about the whole site

Reply:
Yes, we are removing big borders from design